ios/Classes/Blemulator/Bridging/DartMethodCaller.m

```diff
@@ -55,4 +86,21 @@ - (InvokeMethodResultHandler)simpleInvokeMethodResultHandlerForMethod:(NSString
     };
 }
 
+- (InvokeMethodResultHandler)invokeMethodResultHandlerForMethod:(NSString *)methodName
+                                                        onSuccess:(SuccessHandler)successHandler
+                                                         onError:(ErrorHandler)errorHandler {
+    return ^(id _Nullable result) {
+        if ([result class] == [FlutterError class]) {
+            NSLog(@"%@%@%@%@", @"DartMethodCaller.", methodName, @": FlutterError: ", [(FlutterError *)result message]);
```

Comment:
What's %@%@%@?

Reply:
It's for specyfing the format of the arguments in output of NSLog.